### PR TITLE
Meta/linter errors

### PR DESCRIPTION
**Pull request by sulan**
Format, resolve pylint issues, and fix typing errors.

### CODE REVIEW

File: knpy/braid.py [hunk 1]
```diff
@@ -58,90 +63,123 @@ def notation(self, copy = True) -> BraidNotation:
...
+        assumed to be circular, the chunk may cross the end of the array (so some elements from the end, then some
+        elements from the beginning).
+
+        index: Where the chunk starts, on which operation can be done; in the range [-n, n) where n is the number of
+        crossings in the braid (so n = len(braid))
         """
         if self.is_braid_relation1_performable(index):
-            signs = np.ones(3,)
+            signs = np.ones(
+                3,
```
Comment (SzilBalazs) [inline knpy/braid.py:126]: I would suggest removing the coma

File: knpy/data_utils.py [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
+knots_in_braid_notation_dict = {}
+with open(file_path, "r") as file:
+    for i, line in enumerate(file):
+        if i == 0:
+            continue
         line = line.strip()
         line_splitted = line.split(",")
         key = line_splitted[0]
-        temp_braid_notations = [] #TODO Multiple braid notation might be available
+        temp_braid_notations = []  # TODO Multiple braid notation might be available
```
Comment (SzilBalazs) [inline knpy/data_utils.py:16]: W0511: TODO Multiple braid notation might be available (fixme)

Comment (sulan) [reply knpy/data_utils.py:16]: Fixme type warnings are ignored in CI, so I left it here. It's not _trivially_ fixable anyways.

File: knpy/braid.py (hunk 1 above)
Comment (sulan) [reply knpy/braid.py]: Done

**Review by SzilBalazs — APPROVED** [review]
lgtm

**Pull request merged**